backend/src/main/java/com/bakdata/conquery/sql/execution/SqlExecutionService.java

```diff
@@ -76,20 +84,18 @@ private String getColumnName(ResultSet resultSet, int columnIndex) {
 		}
 	}
 
-	private Object[] getResultRow(ResultSet resultSet, int columnCount) {
-		// JDBC ResultSet indices start with 1 and we skip the first column because it contains the id
-		return IntStream.rangeClosed(2, columnCount)
-						.mapToObj(columnIndex -> this.getValueOfColumn(resultSet, columnIndex))
-						.toArray();
-	}
+	private SqlEntityResult getResultRow(ResultSet resultSet, List<ResultType> resultTypes, int columnCount) throws SQLException {
 
-	private String getValueOfColumn(ResultSet resultSet, int columnIndex) {
-		try {
-			return resultSet.getString(columnIndex);
-		}
-		catch (SQLException e) {
-			throw new ConqueryError.SqlError(e);
+		int rowNumber = resultSet.getRow();
+		String id = resultSet.getObject(PID_COLUMN_INDEX).toString();
```

Comment:
Ist der Aufruf anders als dieser?
```suggestion
		String id = resultSet.getString(PID_COLUMN_INDEX);
```

Reply:
Nein, gut gesehen 👍